app/src/main/java/org/oppia/app/player/exploration/ExplorationActivityPresenter.kt

```diff
@@ -14,12 +15,12 @@ class ExplorationActivityPresenter @Inject constructor(private val activity: App
     if (getExplorationFragment() == null) {
       activity.supportFragmentManager.beginTransaction().add(
         R.id.exploration_fragment_placeholder,
-        ExplorationFragment()
+        StateFragment()
       ).commitNow()
     }
   }
 
-  private fun getExplorationFragment(): ExplorationFragment? {
-    return activity.supportFragmentManager.findFragmentById(R.id.exploration_fragment_placeholder) as ExplorationFragment?
+  private fun getExplorationFragment(): StateFragment? {
+    return activity.supportFragmentManager.findFragmentById(R.id.exploration_fragment_placeholder) as StateFragment?
```

Comment:
Revert these changes, we don't need to make sure that the PR code uses your implementation.
Rather than that we want that your PR should be merged directly and whenever the develop runs the develop branch, homefragment should appear.

Reply:
reverted